### PR TITLE
Add library with relative path to .xcodeproj file

### PR DESCRIPTION
**Pull request by Fatme**
Fixes https://github.com/NativeScript/nativescript-cli/issues/679

### CODE REVIEW

File: lib/services/ios-project-service.ts [hunk 1]
```diff
@@ -181,14 +181,15 @@ class IOSProjectService extends projectServiceBaseLib.PlatformProjectServiceBase
...
+			let frameworkName = path.basename(libraryPath, path.extname(libraryPath));
+			let targetPath = path.join("lib", this.platformData.normalizedPlatformName, frameworkName);
+			let fullTargetPath = path.join(this.$projectData.projectDir, targetPath);
+			this.$fs.ensureDirectoryExists(fullTargetPath).wait();
+			shell.cp("-R", libraryPath, fullTargetPath);
 
 			let project = this.createPbxProj();
-
-			project.addFramework(path.join(targetPath, frameworkName + ".framework"), { customFramework: true, embed: true });
+			let frameworkPath = path.relative("platforms/ios", path.join(targetPath, frameworkName + ".framework"));
```
Comment (rosen-vladimirov) [inline lib/services/ios-project-service.ts]: `platforms/ios` - will this work on Windows?
Maybe: 
```TypeScript
let frameworkPath = path.relative(path.join("platforms", "ios"), path.join(targetPath, frameworkName + ".framework"));
```

Comment (rosen-vladimirov) [inline lib/services/ios-project-service.ts]: Also, don't you need full paths on both places:
```TypeScript
let frameworkPath = path.relative(path.join(this.$projectData.projectDir, "platforms", "ios"), path.join(fullTargetPath, frameworkName + ".framework"));
```

Comment (Fatme) [inline lib/services/ios-project-service.ts]: platforms/ios - will this work on Windows? - This service is executed only on OSX.
Also, don't you need full paths on both places: - No, I don't need the fullPath, a just need the relative path to native project.

Comment (rosen-vladimirov) [inline lib/services/ios-project-service.ts]: :+1:

**Comment by rosen-vladimirov** [comment]
:+1:

File: lib/services/ios-project-service.ts (hunk 1 above)
Comment (teobugslayer) [inline lib/services/ios-project-service.ts]: By the way, it works - but path.join will return a string with backslashes.

**Pull request merged**